README.md

```diff
@@ -6,13 +6,14 @@ This is the integration / wayfinding repository for the Unicorn Delivery Service
 - [GitLab Runner](https://github.com/defenseunicorns/uds-package-gitlab-runner) - a Continuous Integration runner that integrates with GitLab
 - [Mattermost](https://github.com/defenseunicorns/uds-package-mattermost) - an open-source, self-hostable online chat service
 - [SonarQube](https://github.com/defenseunicorns/uds-package-sonarqube) - an open-source platform developed by SonarSource for continuous inspection of code quality
+- ⚠️ (alpha) [Coder](https://github.com/defenseunicorns/uds-package-coder) - a cloud IDE solution for developing software without local tooling
```

Comment:
Are you gonna remote this for now based on our other discussion? otherwise looks good
```suggestion
```

Reply:
done